lib/alchemy/resource.rb

```diff
@@ -223,7 +223,7 @@ def engine_name
     def help_text_for(attribute)
       ::I18n.translate!(attribute[:name], scope: [:alchemy, :resource_help_texts, resource_name])
     rescue ::I18n::MissingTranslationData
-      false
+      nil
```

Comment:
Would like to have a spec for this change.

Reply:
Will do

Reply:
Thanks